docs/reference/modules/admin-bar.md

```diff
@@ -14,7 +14,7 @@ This module implements Apostrophe's admin bar at the top of the page. Any module
 
 Options are passed into the admin-bar module by "extending" it through an `index.js` file typically located at `modules/@apostrophecms/admin-bar/index.js in the project folder.
 
-The options passed in this manner will configure the existing @apostrophe/admin-bar` module options through implicit subclassing. This same type of configuration takes place when you create a `modules->@apostrophecms->pages->index.js` file in the project to add configuration to the main `page` module.
+The options passed in this manner will configure the existing @apostrophe/admin-bar` module options through implicit subclassing. This same type of configuration takes place when you create a `modules/@apostrophecms/pages/index.js` file in the project to add configuration to the main `page` module.
```

Comment:
Please reuse the new language in the global module reference page about merging configuration, in place of this language.

Reply:
Changed.